features/argument_transforms/integer.feature

```diff
@@ -1,41 +1,33 @@
-Feature: An argument that could represent an integer will be
-    transformed into an integer type.
+Feature: Integer
+  An argument that could represent an integer will be transformed into an integer type.
 
-  Scenario Outline: Assorted basic inputs are provided.
-    Given a file named "features/transform_integer.feature" with:
-      """
-
-Feature: Transform integer arguments.
-  Scenario: Docstring simple value.
+  Scenario Outline: Docstring simple value.
     When the response body is assigned:
-      \"\"\"
+      """
       <input>
-      \"\"\"
-    Then the response body as JSON is:
-      \"\"\"
-      '<expected>'
-      \"\"\"
+      """
+    Then the value of the response body is a valid `Integer`
+    And the value of the response body is equal to `<input>`
 
-  Scenario: Inline simple value.
-    When the response body is assigned `<input>`
-    Then the response body as JSON is:
-      \"\"\"
-      '<expected>'
-      \"\"\"
+  Examples:
+    |               input |
+    |                   0 |
```

Comment:
what is the expected behavior if say 1.5 or some non-integer was provided?

Reply:
Right now non-integer numeric types are not supported and would be left as strings. Apparently that hasn't come up yet. I just opened #174 to check this out later